packages/react/accordion/src/Accordion.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { getSelector } from '@radix-ui/utils';
 import {
   composeEventHandlers,
-  createContext,
+  createContextObj,
```

Comment:
I'll rename this back to `createContext` in the last PR. Unless we think this is more explicit? The context that the provider provides will always be an object.